go/vt/vtgate/planbuilder/testdata/union_cases.txt

```diff
@@ -1808,11 +1855,15 @@ Gen4 plan same as above
   "Instructions": {
     "OperatorType": "SimpleProjection",
     "Columns": [
-      1
+      2
     ],
     "Inputs": [
       {
         "OperatorType": "Distinct",
+        "Collations": [
+          "1: binary"
+        ],
+        "ResultColumns": 1,
```

Comment:
I think only 1 column will be returned from the input. The Columns on the SimpleProjection contains 2nd index. 
The plan may be incorrect.